dominion-solutions/laravel
# Added php7-pecl-redis to Dockerfile

# Details

Added php7-pecl-redis to Dockerfile.

# Steps to test changes

1. Use ***docker build . -f docker/app/Dockerfile*** to build the image
1. Check there are no errors

# Issues Addressed

[CU-c2j0dx](https://app.clickup.com/t/c2j0dx)
